### PR TITLE
SNHU:1414 - Front-end app scaffolding

### PR DESCRIPTION
**Pull request by joao-sd**
Description
=====

This PR implements basic routing, a user module, user service, and integration with our storybook library.

Note that to install our stories module you'll need a .npmrc that should be asked for in our slack channel.

### CODE REVIEW

File: apps/toboggan-app/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { UserMainPageComponent } from './user/pages/user-main-page/user-main-page.component';
+
```
[the remark this answers is not in the document]
Comment (joao-sd) [reply apps/toboggan-app/src/app/app-routing.module.ts:3]: Done!

File: apps/toboggan-app/src/app/user/user.routing.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { RouterModule, Routes } from '@angular/router';
+import { UserMainPageComponent } from './pages/user-main-page/user-main-page.component';
+
+const userRoutes: Routes = [{ path: '', component: UserMainPageComponent }];
+
+export const userRouting = RouterModule.forChild(userRoutes);
```
Comment (sada-ajith) [inline apps/toboggan-app/src/app/user/user.routing.ts:6]: Case of class name can be updated

File: apps/toboggan-app/src/app/user/services/user.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+describe('UserService', () => {
+  let service: UserService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(UserService);
+  });
+
+  it('should be created', () => {
```
Comment (sada-ajith) [inline apps/toboggan-app/src/app/user/services/user.service.spec.ts:13]: we might need test cases for functions in service

**Review by sada-ajith — COMMENTED** [review]
We might need add test cases for functions inside service . We can probably add it later .

**Review by dmitriy-rodimin-sada — APPROVED** [review]

**Review by sada-ajith — APPROVED** [review]

File: apps/toboggan-app/src/app/user/services/user.service.spec.ts (hunk 3 above)
Comment (joao-sd) [reply apps/toboggan-app/src/app/user/services/user.service.spec.ts:13]: We'll up in a separate ticket then 👍🏽

**Pull request merged**